File: runtimes/wasmer/typescript/baseline.ts
#!/usr/bin/env tsx
/**
 * Baseline TypeScript app without Wasmer - for size comparison
 */

function add(a: number, b: number): number {
  return a + b;
}

function main() {
  const result = add(5, 3);
  console.log(`Result of add(5, 3): ${result}`);
  
  if (result !== 8) {
    throw new Error(`Expected 8, got ${result}`);
  }
  console.log("✓ Test passed!");
}

main();

File: runtimes/wasmer/python/baseline.py
#!/usr/bin/env python3
"""Baseline Python app without Wasmer - for size comparison"""

def add(a, b):
    return a + b

def main():
    result = add(5, 3)
    print(f"Result of add(5, 3): {result}")
    assert result == 8, f"Expected 8, got {result}"
    print("✓ Test passed!")

if __name__ == "__main__":
    main()